spec/prawn/vera_pdf.rb

```diff
@@ -1,4 +1,4 @@
-require 'nokogiri'
```

Comment:
Done. It is better located in `extensions`. No need to require the veraPDF helpers any more. :+1: